tasks/custom-element-json.js

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { execSync } from 'child_process';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import path from 'path';
+
+// Get a list of all packages except those you want to ignore
+const getWorkspacePackages = (ignoredPackages) => {
+    const workspaceInfo = execSync('yarn workspaces info --json').toString();
+    const workspacePackages = JSON.parse(workspaceInfo);
+    return Object.entries(workspacePackages)
+        .filter(([pkgName]) => !ignoredPackages.includes(pkgName))
+        .map(([pkgName, pkgDetails]) => ({
+            name: pkgName,
+            path: pkgDetails.location,
+        }));
+};
+
+const __filename = dirname(fileURLToPath(import.meta.url));
+const __dirname = dirname(__filename);
+const configPath = path.resolve(
+    __dirname,
+    'custom-elements-manifest.config.js'
+);
+
+// Define the packages to ignore
+const ignoredPackages = [
```

Comment:
If these are the same ignored packages for `tasks/confirm-build.js`, `tasks/custom-element-json.js`, `tasks/remove-custom-elements-json.js`, `tasks/run-check-cem.js`, and `tasks/gen-react-wrapper.js`, could we make `getWorkspacePackages` and `ignoredPackages` part of a separate module that these scripts can import. That way we don't accidentally miss updating one of them if we update the ignored packages?

Reply:
done!

Reply:
Apparently the ignoredPackages for all these packages might look identical but it is not.
For instance `reactive-controller` is not present in some files in ignoredPackages.